watson-developer-cloud/discovery-components
# [WIP] HtmlViewer- Create new component (and sub-components) and a basic story

#### What do these changes do/fix?

Add a component to be used for CI documents, enrichments view, and simple document view (document preview).

https://github.ibm.com/Watson-Discovery/the-tooling-core-tracker/issues/1259

#### How do you test/verify these changes?

Open Storybook and look at the HtmlViewer story. Click through the knobs to see different types of documents.

#### Have you documented your changes (if necessary)?

Yes, in Storybook.

#### Are there any breaking changes included in this pull request?

No.
